bin/cfg/TR1X_gameflow.json5

```diff
@@ -152,6 +157,7 @@
             ],
             "sequence": [
                 {"type": "play_fmv", "fmv_path": "fmv/lift.avi"},
+                {"type": "loading_screen", "picture_path": "data/images/greece_saturn.png", "display_time": 5},
```

Comment:
Is it intended that Folly has the Saturn image while the rest of Greece doesn't?

Reply:
Oh yes I forgot to ask about this part. POSIX suggested using this loading screen once for St Francis Folly as a Saturn Easter egg. I could also leave it as an alternative in the repo like the PS1 credits.

Reply:
Yes let's go with PS1 version and leave Saturn as an easter egg.